components/ui/Gnb.vue

```diff
@@ -50,36 +50,46 @@ export default {
     },
     async mounted() {
         try {
-
-        await this.$api.season.getSeasonList().then(res => {
-
-            res.forEach((item, index) => {
-                if (item.num) {
-                    item.title = "시즌" + item.num;
-                }
-                if (res[index + 1]) {
-                    if (item.name.indexOf('Tutorial') > 0) {
-                        item.title = "시즌" + item.num + " 설명서";
-                        item.num += 0.5;
-                    } else {
-                        // 겹치면 0.5 시즌
-                        if (res[index + 1].num === item.num) {
+            await this.$store.dispatch('getSeasonList').then(res => {
+                res.forEach((item, index) => {
+                    if (item.num) {
+                        item.title = "시즌" + item.num;
+                    }
+                    if (res[index + 1]) {
+                        if (item.name.indexOf('Tutorial') > 0) {
+                            item.title = "시즌" + item.num + " 설명서";
                             item.num += 0.5;
+                        } else {
+                            // 겹치면 0.5 시즌
+                            if (res[index + 1].num === item.num) {
+                                item.num += 0.5;
+                            }
                         }
                     }
-                }
-                if (item.name.indexOf('Stage') > 0) {
-                    item.title = "시즌" + item.num + "";
-                }
-            });
+                    if (item.name.indexOf('Stage') > 0) {
+                        item.title = "시즌" + item.num + "";
+                    }
+                });
+                this.$store.commit('setSeasonList', res);
 
-            this.$store.commit('setSessionList', res);
-            this.$store.commit('setCurrentSeason', this.currentSeason);
-        });
+                let seasonInfo = res.filter(res => {
+                    if (res.num === Number(this.currentSeason)) {
+                        return res;
+                    }
+                })[0];
+
+                this.$store.commit('setSeasonInfo', seasonInfo);
+                this.$store.commit('setCurrentSeason', this.currentSeason);
+            });
         } catch (e) {
             console.log(e);
         }
     },
+    computed: {
+        seasons() {
+            return this.$store.getters['seasons'];
```

Comment:
Codacy has a fix for the issue: [Replace `'seasons'` with `"seasons"`](https://app.codacy.com/gh/Eight-Corner/front_toche_web/pullRequest?prid=9689045)
```suggestion
            return this.$store.getters["seasons"];
```